docs/03_stratum1_proxies.md

```diff
@@ -366,13 +366,14 @@ CVMFS_HTTP_PROXY="http://<PROXY_IP>:3128"
 
 ***Replace the ``<PROXY_IP>`` part with the IP address of your Squid proxy server!***
 
-After changing the local configuration file, make sure to reload the CernVM-FS client configuration:
+After changing the local configuration file, unmount and probe the repository to enable the new configuration:
 
 ```bash
-sudo cvmfs_config reload repo.organization.tld
+sudo cvmfs_config umount repo.organization.tld
+sudo cvmfs_config probe repo.organization.tld
 ```
 
-More proxies can be added to that list by separating them with a pipe symbol.
+More proxies can be added to `CVMFS_HTTP_PROXY` by separating them with a pipe symbol.
```

Comment:
```suggestion
More proxies can be added to `CVMFS_HTTP_PROXY` by separating them with a pipe symbol (`|`).
```

Reply:
I think/assume it automatically reloads the configuration if you access (and, hence, mount) an unmounted repository (?). But I indeed also considered to let them first do an `ls`, and then do `reload`, which should work as well.

Reply:
Changed it to the latter: first do an `ls` on the repo to make sure it's mounted, then run `reload`.